src/index.ts

```diff
@@ -168,7 +170,8 @@ export class GoogleToken {
         sub: this.sub,
         key: this.key,
         keyFile: this.keyFile,
-        scope: this.scope
+        scope: this.scope,
+        additionalClaims: this.additionalClaims
```

Comment:
Nit. If you put a comma at the end of the last item, you wouldn't have to change that line when you append another item later.

Reply:
That sounds like the kind of thing we should add to the linter if we want to enforce it :)

Reply:
Having been said, fixed